backend/vehicles/src/modules/permit/permit.controller.ts

```diff
@@ -60,6 +60,20 @@ export class PermitController {
     return this.permitService.create(createPermitDto);
   }
 
+  @ApiCreatedResponse({
+    description: 'The Permit Resource',
+    type: ReadPermitDto,
+    isArray: true,
+  })
+  @Public()
+  @Get()
+  async get(
+    @Req() request: Request,
```

Comment:
Can `request` be removed? I don't think it's used anywhere.

Reply:
left it incase we needed it when we implement auth, I'll take it out for now